### PR TITLE
Jelares MVP Front End

### PR DESCRIPTION
**Pull request by jelares**

### CODE REVIEW

File: backstory/src/main/java/com/google/sps/story/PromptManager.java [hunk 1]
```diff
@@ -11,7 +11,7 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
-package com.google.sps.servlets;
+package com.google.sps.story;
```
Comment (jelares) [inline backstory/src/main/java/com/google/sps/story/PromptManager.java:14]: Changed for ease of interfacing with the backend.

File: backstory/src/test/java/com/google/sps/story/PromptManagerTest.java [hunk 2]
```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package com.google.sps.servlets;
+package com.google.sps.story;
```
Comment (jelares) [inline backstory/src/test/java/com/google/sps/story/PromptManagerTest.java:15]: Changed for consistency and ease of interfacing with the backend.

File: backstory/src/main/webapp/index.html [hunk 3]
```diff
@@ -3,6 +3,15 @@
   <head>
     <meta charset="UTF-8" />
     <title>Home</title>
+
+    <!-- Bootsrap Stylesheet -->
```
Comment (aaronegolden) [inline backstory/src/main/webapp/index.html:7]: Nit: Might be good to including a link to the license in the comment. It's MIT licensed, which is fine, but it's hard to tell without inspecting the downloaded file right now:

https://github.com/twbs/bootstrap/blob/main/LICENSE

**Comment by UdayKalra** [comment]
Address the JavaScript line length validation failure before merging.

File: backstory/src/main/webapp/index.html (hunk 3 above)
Comment (jelares) [reply backstory/src/main/webapp/index.html:7]: Sounds good - updated!

**Pull request merged**